### ISSUE TITLE
Some endpoints return arrays but are defined as objects

### ISSUE THREAD

**Issue by vitld** [s-crypt/IGDB-OpenAPI#7]
Hi,

First of all, thanks for maintaining this spec. It simplifies and saves a lot of time when setting up the service layer in an app.

I'm not even sure this a bug or issue, but with my limited knowledge of OpenAPI there seems to be some error in the defs. Feel free to tell me I'm wrong.

I just recently started using the IGDB API on a hobby project and decided to use the OpenAPI TypeScript(https://openapi-ts.dev/) library together with OpenAPI Fetch. Converting the OpenAPI yaml works fine but pretty shortly started to notice that the responses from endpoints did not match the types generated by OpenAPI TypeScript. Mainly that the responses contained arrays of objects, while the generated response typescript type was a plain object.

Will be using the `/games` endpoint as example from now on, with redundant fields removed.

Current OpenAPI spec:
```yaml
  /games:
      responses:
        "200":
          description: Success
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Games"
```

Results in the following type generated:
```typescript
    RetreiveGames: {
        responses: {
            /** @description Success */
            200: {
                headers: {
                    [name: string]: unknown;
                };
                content: {
                    "application/json": components["schemas"]["Games"];
                };
            };
        };
    };
```

Which results in incorrect types, because the response, as mentioned, contains array of games.

If I change the OpenAPI yaml to the following:
```yaml
      responses:
        '200':
          description: Success
          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/Games'
```
Then I get correct types:
```typescript
        responses: {
            /** @description Success */
            200: {
                headers: {
                    [name: string]: unknown;
                };
                content: {
                    "application/json": components["schemas"]["Games"][];
                };
            };
        };

```
Notice: 
`"application/json": components["schemas"]["Games"]` vs ` "application/json": components["schemas"]["Games"][]`.

Is this an issue or am I not understanding OpenAPI correctly?

**Comment by s-crypt** [comment]
You would be correct! I guess I missed the opening and closing square brackets when checking responses. Oops. Thanks for taking the time to document and bring this up!

I used the following regex to correct each path to specify that an array is being returned:
Find: `(applicati.+)(\n.+)\n( +)(.+)`
Replace: `$1$2\n$3type: array\n$3items:\n$3  $4`

I don't have the time to check each endpoint to see if it returns an array of objects right now, but I think they all do. This change is made in #8. Let me know if it works and I will push it to master!

**Comment by s-crypt** [comment]
Resolved with #8
